interface/code_systems/standard_tables_manage.php

```diff
@@ -83,6 +84,13 @@
             exit;
          }
     }
+    if ($version == "RF2International:English") {
```

Comment:
Actually, would make more sense to make this more flexible and to do the following.
1. Pass the if here if the $version starts with RF2
2. After passing the if, then remove the RF2 from the beginning of $version
(this will allow flexibility if using other RF2 packages, such as the US release which will import the same exact way as the international version)